Neolution.DotNet.Console/Abstractions/IDotNetConsoleCommand.cs

```diff
@@ -12,7 +13,8 @@ public interface IDotNetConsoleCommand<in TOptions>
         /// Runs the command with the specified options asynchronously.
         /// </summary>
         /// <param name="options">The options.</param>
+        /// <param name="cancellationToken">The cancellation token.</param>
         /// <returns>The <see cref="Task"/>.</returns>
-        Task RunAsync(TOptions options);
+        Task RunAsync(TOptions options, CancellationToken cancellationToken);
```

Comment:
maybe have both, with and without cancellation token, and have a default implementation that the one without the cancellation token calls the other with CancellationToken.None

Reply:
This doesn't prevent the breaking change anyway, upgrading you still have to implement the new interface method with the cancellation token